src/lib/components/jam/modals/Settings.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+    import Button from '@lib/components/global/Button.svelte';
+    import Icon from '@lib/components/global/Icon.svelte';
+    import Modal from '@lib/components/global/Modal.svelte';
+
+    export let closeFunc: Function;
+</script>
+
+<Modal
+    name="settings"
+    {closeFunc}
+    ><div class="settings">
+        <div class="categories">
+            <Button type="button"
+                ><Icon name="speaker" />
+                <p>Playback</p></Button>
+            <Button type="button"
+                ><Icon name="layout" />
+                <p>Playback</p></Button>
+        </div>
+        <div class="content" />
+    </div>
+</Modal>
+
+<style lang="scss">
+    .settings {
+        width: 25rem;
+        padding: 1rem;
```

Comment:
This padding looks strange to me when the Modal title text has no extra padding
```suggestion
```

#### With padding:
<img width="579" alt="image" src="https://user-images.githubusercontent.com/9354822/220273217-f534c95b-0bc4-438a-afea-9e089cd717be.png">

#### Without:
<img width="530" alt="image" src="https://user-images.githubusercontent.com/9354822/220273087-001e01fa-b7e2-42b6-81fa-95e1c80829a4.png">

Reply:
This modal is not ready BTW but yes that's good.